dev-games/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="Simulator Construction Toolkit"
+DESCRIPTION="Set of open-source libraries designed as building blocks \
+for quickly assembling 3D simulations, games, and visualization applications."
+HOMEPAGE="https://home.flightgear.org/"
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+PATCHES="simgear-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	simgear$secondaryArchSuffix = $portVersion
+	lib:libSimGearCored$secondaryArchSuffix = $portVersion
+	lib:libSimGearScened$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libosg$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	simgear$secondaryArchSuffix = $portVersion
+	devel:libSimGearCored$secondaryArchSuffix = $portVersion
+	devel:libSimGearScened$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	simgear$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_system$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libosg$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. $cmakeDirArgs \
+		-DCMAKE_INSTALL_PREFIX:PATH="$prefix" \
+		-DCMAKE_INSTALL_INCLUDEDIR:PATH="$includeDir" \
+		-DCMAKE_INSTALL_LIBDIR:PATH="$libDir" \
+		-DSIMGEAR_SHARED:BOOL=ON \
+		-DENABLE_TESTS:BOOL=OFF
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cmake -P cmake_install.cmake
+	rm -f "$libDir"/*.la
+	mkdir -p `dirname $includeDir` $libDir
+	mv $prefix/include $includeDir
```

Comment:
@kenmays, I'm trying this right now:
```diff
 INSTALL()
 {
+	cd build
 	cmake -P cmake_install.cmake
-	rm -f "$libDir"/*.la
-	mkdir -p `dirname $includeDir` $libDir
-	mv $prefix/include $includeDir
```
with a `PATCH` function, before `BUILD`:
```
	"

PATCH()
{
	sed -i \
		-e "/^install .*DESTINATION include/ \
			s|include\(/simgear/\)|$relativeIncludeDir\1|" \
		CMakeLists.txt \
		simgear/CMakeLists.txt
}

BUILD()
```